src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1886,6 +1886,44 @@ namespace UnitarySystems {
 
             SizingMethod = DataHVACGlobals::SystemAirflowSizing;
 
+            MSHPIndex = this->m_DesignSpecMSHPIndex;
+            if (MSHPIndex > -1) {
+                // set no load air flow ratio local var
+                Real64 NoLoadCoolingAirFlowRateRatio = 1.0;
+                Real64 NoLoadHeatingAirFlowRateRatio = 1.0;
+                if (designSpecMSHP[MSHPIndex].coolingVolFlowRatio[0] == DataSizing::AutoSize) {
+                    NoLoadCoolingAirFlowRateRatio = 1.0 / double(designSpecMSHP[MSHPIndex].numOfSpeedCooling);
+                } else {
+                    NoLoadCoolingAirFlowRateRatio = min(this->m_NoLoadAirFlowRateRatio, designSpecMSHP[MSHPIndex].coolingVolFlowRatio[0]);
+                }
+                if (designSpecMSHP[MSHPIndex].heatingVolFlowRatio[0] == DataSizing::AutoSize) {
+                    NoLoadHeatingAirFlowRateRatio = 1.0 / double(designSpecMSHP[MSHPIndex].numOfSpeedHeating);
+                } else {
+                    NoLoadHeatingAirFlowRateRatio = min(this->m_NoLoadAirFlowRateRatio, designSpecMSHP[MSHPIndex].heatingVolFlowRatio[0]);
+                }
+                this->m_NoLoadAirFlowRateRatio = min(NoLoadCoolingAirFlowRateRatio, NoLoadHeatingAirFlowRateRatio);
+            }
```

Comment:
Then add an else here and use those same 2 variables.

Reply:
Done